src/mare/source.cr

```diff
@@ -63,6 +63,22 @@ struct Mare::Source::Pos
     new(source, start, start, line_start, line_finish, row, col)
   end
 
+  def self.point(source : Source, offset : Int32)
+    line_start = 0
+    line_finish = source.content.index("\n") || source.content.size
+    row = 0
+
+    while line_finish < offset
```

Comment:
oh, this will loop forever if `offset` is bigger than `source.content.size`. Lemme fix that

Reply:
Fixing with https://github.com/jemc/mare/pull/68